### ISSUE TITLE
is it possible to fetch movies from google td?

### ISSUE THREAD

**Issue by askfriends** [josprojects/tgmoviebot#3]
Hi;
is it possible to fetch movies from google drive and share the links in the group when requested by any user?
If not than can you add that feature?

**Comment by MR-JINN-OF-TG** [comment]
Can't fetch movies from Google but here that it iave spewing mode for so you can search it correctly. And this bot giving movies vai it's a database alrwdy saved from channels on telegram.

**Comment by askfriends** [comment]
> Can't fetch movies from Google but here that it iave spewing mode for so you can search it correctly. And this bot giving movies vai it's a database alrwdy saved from channels on telegram.

yeah but there is gdrive api which can be used to fetch movies from gdrive or tds, there are already many bots available which dos the same job, can you have a look and see if you can implement that in your repo?

**Comment by MR-JINN-OF-TG** [comment]
> > Sorry Its not mine repo. Ask the devs of the respository for more.
